bin/docker-image-tool.sh

```diff
@@ -44,15 +44,37 @@ function image_ref {
 function build {
   local BUILD_ARGS
   local IMG_PATH
+  local TMPFOLDER
 
   if [ ! -f "$SPARK_HOME/RELEASE" ]; then
     # Set image build arguments accordingly if this is a source repo and not a distribution archive.
+    local JARS="${SPARK_HOME}/assembly/target/scala-${SPARK_SCALA_VERSION}/jars"
+    TMPFOLDER=`mktemp -q -d examples.XXXXXX`
```

Comment:
This will probably work, just wondering if we can use `-p /some/path` where `/some/path` is guaranteed to exist. This could fail in theory if for some reason `/tmp` doesn't exist (or doesn't have space?) or if the user's TMPDIR is set to something non-writable